src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -5,34 +5,49 @@
 
 class BreadcrumbsHelper extends CoreBreadcrumbsHelper
 {
+    use OptionsAwareTrait;
     /**
      * @inheritdoc
      */
     protected $_defaultConfig = [
         'templates' => [
-            'wrapper' => '<ol{{attrs}}>{{content}}</ol>',
-            'item' => '<li{{attrs}}><a href="{{url}}"{{innerAttrs}}>{{title}}</a></li>{{separator}}',
-            'itemWithoutLink' => '<li{{attrs}}><span{{innerAttrs}}>{{title}}</span></li>{{separator}}',
-            'separator' => '<li{{attrs}}><span{{innerAttrs}}>{{separator}}</span></li>'
+            'wrapper' => '<nav aria-label="breadcrumb"><ol{{attrs}}>{{content}}</ol></nav>',
+            'item' => '<li{{attrs}}><a href="{{url}}"{{innerAttrs}}>{{title}}</a></li>',
+            'itemWithoutLink' => '<li{{attrs}} aria-current="page"><span{{innerAttrs}}>{{title}}</span></li>',
+            'separator' => ''
         ]
     ];
 
     /**
-     * Default attributes for the wrapper
+     * Default attributes for the templates
      *
      * @var array
      */
     protected $_defaultAttributes = [
-        'class' => 'breadcrumb'
+        'class' => [
+            'wrapper' => 'breadcrumb',
+            'item' => 'breadcrumb-item',
+            'itemWithoutLink' => 'breadcrumb-item active'
```

Comment:
This doesn't seem to be used anywhere, should it be?

Reply:
Default templates doesn't include the class anymore. The above classes are injected in the [render](https://github.com/julianpollmann/bootstrap-ui/blob/feature/bootstrap4/src/View/Helper/BreadcrumbsHelper.php#L39) and [add](https://github.com/julianpollmann/bootstrap-ui/blob/feature/bootstrap4/src/View/Helper/BreadcrumbsHelper.php#L49) method.